web-common/src/features/dashboards/dashboard-stores.ts

```diff
@@ -492,14 +492,24 @@ const metricViewReducers = {
 
   displayDeltaChange(name: string) {
     updateMetricsExplorerByName(name, (metricsExplorer) => {
-      // NOTE: only show delta change if comparison is enabled
+      // NOTE: only show delta change if time comparison is enabled
       if (metricsExplorer.showComparison === false) return;
 
       metricsExplorer.leaderboardContextColumn =
         LeaderboardContextColumn.DELTA_PERCENT;
     });
   },
 
+  displayDeltaAbsolute(name: string) {
```

Comment:
I am not sure if having a dedicated method for every option makes sense. Could we perhaps take the LeaderboardContextColumn value as argument?